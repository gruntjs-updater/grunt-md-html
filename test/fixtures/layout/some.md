Test
====

```javascript
var a = 10;
```

下面是个表格

| key | value |
| --- | ---   |
| name | helinjiang |