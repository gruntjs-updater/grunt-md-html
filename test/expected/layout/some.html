<!DOCTYPE html>
<html>

<head>
    <title></title>
    <style>
        body {
            color: #fff;
        }
        
        .s {
            color: red
        }
    </style>
</head>

<body>
    <p>test/fixtures/layout_api/some.md</p>
    <p>tmp/layout/some.html</p>
    <p>test/fixtures/layout_api/some.md</p>
    <p>hello world!</p>
    <p>&lt;!----&gt;
    </p>
    <p>&lt;!-- title: Object.prototype date: 2015-11-21--&gt;
    </p>
    <h1 id="some">some</h1>
    <p>I am in layout_api some.md</p>


    <script>
        var a = 1;
        var b = 3;
        console.log(a, b);
    </script>
</body>

</html>
<!-- build in 2015-12-27 23:00:27-->